sql_queries/Create_SpatialSubsetTable.sql

```diff
@@ -0,0 +1,26 @@
+﻿SET SEARCH_PATH = marvl3, public;
+
+DROP TABLE IF EXISTS spatial_subset;
+CREATE TABLE spatial_subset (
+	source_id integer,
+	origin_id text,
+	"LONGITUDE" double precision,
+	"LONGITUDE_QC" text,
+	"LATITUDE" double precision,
+	"LATITUDE_QC" text,
+	"TIME" timestamp with time zone,
+	"TIME_QC" text,
+	"DEPTH" double precision,
+	"DEPTH_QC" text,
```

Comment:
Don't know if that can affect memory but QC ancillary variables can be store in byte, whether they are a number or a letter (if only one letter).